index.js

```diff
@@ -76,5 +76,7 @@ module.exports = {
   // Local Logistic Regression object for local predictions
   LocalLogisticRegression: require('./lib/LocalLogisticRegression'),
   // Local Logistic Regression object for local predictions
```

Comment:
comment should be changed to the correct local resource name. Probably done if you previously rebase to my topic branch